substrate/frame/offences/src/lib.rs

```diff
@@ -73,14 +73,28 @@ pub mod pallet {
 
 	/// The primary structure that holds all offence records keyed by report identifiers.
 	#[pallet::storage]
-	#[pallet::getter(fn reports)]
 	pub type Reports<T: Config> = StorageMap<
 		_,
 		Twox64Concat,
 		ReportIdOf<T>,
 		OffenceDetails<T::AccountId, T::IdentificationTuple>,
 	>;
 
+	impl<T: Config> Pallet<T> {
```

Comment:
There's an existing impl block elsewhere in this file that you can combine this with